.github/workflows/nightlies.yml

```
@@ -0,0 +1,52 @@
+name: Upload nightly wheels to Anaconda Cloud
+
+on:
+  # Run daily at 1:23 UTC to upload nightly wheels to Anaconda Cloud
+  schedule:
+  - cron: '23 1 * * *'
+  # Run on demand with workflow dispatch
+  workflow_dispatch:
+
+jobs:
+  upload_nightly_wheels:
+    name: Upload nightly wheels to Anaconda Cloud
+    runs-on: ubuntu-latest
```

We probably don't want to run this on forks, so should add a condition here:
```suggestion
    runs-on: ubuntu-latest
    if: github.repository_owner == 'matplotlib'
```

---

Forks won't have access to the secret at least.

---

Yeah, this makes sense. We do this on certain workflows for pyhf with `github.repository == 'scikit-hep/pyhf'`.